### ISSUE TITLE
 graphic (and text) are partly hidden

### ISSUE THREAD

**Issue by Laubeee** [arXiv/html_feedback#1770]
### Description

graphic (and text) are partly outside the visible portion, even with the scrollbar

### (Optional:) Please add any files, screenshots, or other information here.

![image](https://github.com/user-attachments/assets/adee3158-3188-45a7-9213-2811f87d097c)

### (Required) What is this issue most closely related to? Select one.

Figures

### Internal issue ID

07ffe1f4-ed94-4e55-a8bc-55d420d72e86

### Paper URL

https://arxiv.org/html/2312.15234v1

### Browser

Chrome/126.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.

**Comment by Laubeee** [comment]
@dginev thanks, I just checked - yes the graphic is indeed better now. However some text is now rendered black (in dark mode)
I am assuming this is already reported elsewhere - if you need me to open a new issue, let me know.


<img width="383" height="459" alt="Image" src="https://github.com/user-attachments/assets/9e8a47d2-fba4-4d77-ae76-9d98034bdfff" />

**Comment by dginev** [comment]
Yes, very well-reported, there are some plans underway to also improve on the hardcoded colors. 

Thanks for pointing it out and definitely feel encouraged to report more issues, especially if you haven't encountered them before.
